Tests/test_file_icns.py

```diff
@@ -4,6 +4,12 @@
 
 import sys
 
+# The the default version of PyPy in Trusty is PyPy 5.4.1.
+# There is a known error in it: TypeError: PyPy does not yet implement the new buffer interface (issue #2163).
+
+SKIP_5_4_1_PYPY = hasattr(sys, 'pypy_version_info') and (
+    sys.pypy_version_info >= (5, 4, 1, 'final', 0))
```

Comment:
Lets make this a little tighter -- either just 5.4.1 or 5.4.0 < v <5.4.1. 

The default case should be that we have to reaffirm this skip for each pypy that fails, not that we never run these tests again.